go.mod

```diff
@@ -2,35 +2,30 @@ module github.com/anz-bank/sysl
 
 go 1.12
 
+replace github.com/spf13/afero => github.com/nofun97/afero v1.2.3
```

Comment:
whats going on here?

Reply:
The previous bugs with windows tests are caused by afero. Their MemMapFs doesn't really support windows because they don't properly add windows root (Windows volume + `/`), that's why we encountered the error `open c:\: file not found`

Reply:
For now, it just refers to the forked module, if they accept my PR, we can just remove the `replace` line